module/VuFindConsole/src/VuFindConsole/Controller/UtilController.php

```diff
@@ -282,7 +282,9 @@ public function sitemapAction()
             ->get(\VuFind\Config\PluginManager::class);
         $generator = new Sitemap(
             $this->serviceLocator->get(\VuFind\Search\BackendManager::class),
-            $configLoader->get('config')->Site->url, $configLoader->get('sitemap')
+            $this->serviceLocator->get(\VuFindSearch\Service::class),
```

Comment:
I wonder if it's worth setting up a factory for the Sitemap class rather than doing all this work inline in the controller. Certainly not a requirement right now, but maybe a change worth considering while we're touching things anyway.

Reply:
...and this was done a couple of weeks ago.